File: common/k2b6s9j/BoatCraft/item/boat/vanilla/wood/Spruce.java
package k2b6s9j.BoatCraft.item.boat.vanilla.wood;

import k2b6s9j.BoatCraft.entity.boat.EntityCustomBoat;
import k2b6s9j.BoatCraft.item.boat.ItemCustomBoat;
import net.minecraft.item.ItemStack;
import net.minecraftforge.oredict.OreDictionary;
import cpw.mods.fml.common.registry.GameRegistry;

public class Spruce extends ItemCustomBoat {
	
	public static int ID;
	public final int shiftedID = this.itemID;

	public Spruce(int par1) {
		super(par1);
		setUnlocalizedName("boatSpruce");
        func_111206_d("boatcraft:boatSpruce");
    	GameRegistry.registerItem(this, "Spruce Wood Boat");
    	OreDictionary.registerOre("itemBoat", new ItemStack(this));
	}
}

File: src/main/scala/k2b6s9j/boatcraft/minecraft/compatibility/vanilla/materials/wood/Spruce.scala
package k2b6s9j.boatcraft.compatibility.vanilla.boat.wood

import k2b6s9j.boatcraft.core.traits.Material
import net.minecraft.util.ResourceLocation

trait Spruce extends Material {

	override def texture: ResourceLocation = 
		new ResourceLocation("boatcraft",
				"textures/entity/boat/vanilla/wood/spruce.png")
	override def name: String = "Spruce"

}